app/views/students/status_updates/index.html.slim

```diff
@@ -1,5 +1,9 @@
 h1 Listing Status Updates
 
+= link_to 'New Status Update', [:new, :students, :status_update], class: 'btn btn-default'
```

Comment:
This should use `new_students_status_update_path` (or whatever it's called, see `rake routes` for the exact name) instead of the Array.